### PR TITLE
Address EE2 compliance issue for AQMv7

### PR DESCRIPTION
**Pull request by JianpingHuang-NOAA**
Add "set -x" to the util sh scripts as required by NCO 

run_nco_combine_ant_bio.sh

### CODE REVIEW

File: utils/run_nexus_hera.ksh [hunk 1]
```diff
@@ -1,4 +1,7 @@
-#!/bin/ksh -x
+#!/bin/bash
```
Comment (zmoon) [inline utils/run_nexus_hera.ksh:1]: Should the file extension be changed as well?

Comment (JianpingHuang-NOAA) [reply utils/run_nexus_hera.ksh:1]: A good suggestion, it was updated

**Comment by bbakernoaa** [comment]
@zmoon do you approve of the changes?

**Review by zmoon — APPROVED** [review]

**Comment by bbakernoaa** [comment]
@JianpingHuang-NOAA please update the hash in the workflow to use [0b247f5](https://github.com/noaa-oar-arl/NEXUS/commit/0b247f5a996b9b55d5d93f91bfa4220ad2e3ad14)

**Comment by JianpingHuang-NOAA** [comment]
Thanks !

Jianping

**Pull request merged**